src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  margin: unset;
+}
+
+body div {
+  display: flex;
+}
```

Comment:
Related to previous comment
```suggestion
.stars {
  display: flex;
}
```